rules/windows/defense_evasion_clearing_windows_security_logs.toml

```diff
@@ -0,0 +1,27 @@
+[metadata]
+creation_date = "2020/11/12"
+ecs_version = ["1.6.0"]
```

Comment:
It is no longer required to define this unless validation is desired from something other than the _latest_.  Ref: https://github.com/elastic/detection-rules/pull/570

```suggestion
```